packages/sdk/src/swap/sdk.ts

```diff
@@ -145,6 +145,8 @@ export class SwapSDK {
       depositChannelId: response.id,
       depositAddress: response.depositAddress,
       brokerCommissionBps: response.brokerCommissionBps,
+      boostFeeBps:
+        response.boostFeeBps || depositAddressRequest.boostFeeBps || 0,
```

Comment:
should we add this to the response of `requestSwapDepositAddress` called above as well?

Reply:
wdym here? the rpc response doesn't include the `boost_fee` in it.

Reply:
okay lol then its already consistent at least 😅

Reply:
okay we include the input value in the `response` object we assign in the block above, thats why i thought it is returned from the broker. might make sense to change that to be consistent